pkgs/development/libraries/magic-enum/default.nix

```
@@ -15,7 +15,10 @@ stdenv.mkDerivation rec{
 
   nativeBuildInputs = [ cmake ];
 
-  doCheck = true;
+  doCheck = false; # disabled test until upstream build issiues with gcc 12 see https://github.com/Neargye/magic_enum/issues/235
```

```suggestion
  # disable tests until upstream fixes build issues with gcc 12
  # see https://github.com/Neargye/magic_enum/issues/235
  doCheck = false;
```

Also, `issius` -> `issues` in the commit message.